SimplePayment.Test/UnitTest1.cs

```diff
@@ -7,12 +12,67 @@ public class Tests
         [SetUp]
         public void Setup()
         {
+            TestBillingDetailModel();
+            TestPaymentResponseModel();
+        }
+        public string RemoveWhiteSpace(string json)
```

Comment:
Az ilyen segéd method az lehet private, és általában az osztály aljára érdemes rakni, de ez preferencia kérdése. Én személy szerint szeretem őket lentebbre rakni, mert először ha megnyitok egy test file-t, a tesztek érdekelnek leginkább, és ha valami helper method van, abba nem is feltétlenül nézek bele, szóval olvashatóság miatt nekem úgy hatákonyabb.